src/Grid/Service/FavoriteService.php

```diff
@@ -68,4 +79,51 @@ public function removeAssetConfigurationAsFavoriteForCurrentUser(
 
         return $gridConfiguration;
     }
+
+    private function isCurrentUserAllowsToSetAsFavorite(GridConfiguration $gridConfiguration): bool
+    {
+        if ($gridConfiguration->getOwner() === $this->securityService->getCurrentUser()->getId()) {
```

Comment:
Would it not be better to get the current user once and then pass him to private methods?

Reply:
Good Point! I will change this :)

Reply:
Fixed here https://github.com/pimcore/studio-backend-bundle/pull/386/commits/70d12f4a443291c090f59c26b4a06c76097383ba